public/docs/ts/latest/tutorial/toh-pt6.jade

```diff
@@ -229,7 +229,7 @@ block get-heroes-details
   ## Update hero details
 
   We can edit a hero's name already in the hero detail view. Go ahead and try
-  it. As we type, the hero name is updated in the view heading. 
+  it. As we type, the hero name is updated in the view heading.
```

Comment:
We are switching ot _you_. But this switch is just one part of a deliberate sweep of the docs leading to conformance with the Google docs standards. No need for Brandon to take that on for _this_ particular revision.

_After_  a doc has been converted, _then_ our revisions must conform.